packages/editor/ProseMirror/utils/adjustListItemDropInsertPos.test.ts

```
@@ -0,0 +1,108 @@
+
+	tr.replaceRange(mappedInsertPos, mappedInsertPos, slice);
+	return tr.doc;
+};
+
+const listItemTexts = (listNode: ProseMirrorNode) => {
+	const result: string[] = [];
+	listNode.forEach(node => {
+		if (node.type === schema.nodes.list_item) {
+			result.push(node.textContent);
+		}
+	});
```

ESLint

[error] Prefer for...of instead of Array.forEach

(github/array-foreach)